Makefile

```diff
@@ -139,7 +139,8 @@ doc/_build/html:
 	cd doc && make html
```

Comment:
Do we still need `doc/_build/html` target?

Reply:
Yes -- so that we can ensure that it is present when doing `make open-doc`. That said, I streamlined it a bit so there's less duplication and we now don't have the manual directory changes. (Thanks for the Makefile hint!)